lib/tapioca/gem/listeners/yard_doc.rb

```diff
@@ -17,6 +17,7 @@ class YardDoc < Base
           "shareable_constant_value:",
           "rubocop:",
           "@requires_ancestor:",
+          ": ",
```

Comment:
This is used with an `include?` call down below. I want to make sure I'm not ignoring more comments than necessary so added the space.

Reply:
I think prefer a fully fledge `rbs_comment?` method we call from `documentation_comments`.

Or at least a comment explaining what is this `: `:
```suggestion
          ": ", # ignore RBS signature comments
```